test/e2e/antreapolicy_test.go

```diff
@@ -1643,48 +1643,88 @@ func testANNPNestedGroupCreateAndUpdate(t *testing.T, data *TestData) {
 	executeTestsWithData(t, testCase, data)
 }
 
-// testBaselineNamespaceIsolation tests that an ACNP in the baseline Tier is able to enforce default namespace isolation,
+// testACNPBaselineIsolation tests that an ACNP in the baseline Tier is able to enforce default isolation,
 // which can be later overridden by developer K8s NetworkPolicies.
-func testBaselineNamespaceIsolation(t *testing.T) {
-	builder := &ClusterNetworkPolicySpecBuilder{}
-	nsExpOtherThanX := metav1.LabelSelectorRequirement{
-		Key:      "ns",
+func testACNPBaselineIsolation(t *testing.T) {
+	builder1 := &ClusterNetworkPolicySpecBuilder{}
+	podExpOtherThanC := metav1.LabelSelectorRequirement{
+		Key:      "pod",
 		Operator: metav1.LabelSelectorOpNotIn,
-		Values:   []string{getNS("x")},
+		Values:   []string{"c"},
 	}
-	builder = builder.SetName("acnp-baseline-isolate-ns-x").
+	builder1 = builder1.SetName("acnp-baseline-isolate-pod-c").
```

Comment:
I think the policy still isolates ns-x instead of pod-c?

Reply:
I modified the selector expression to be podSelector for pod C, and moved ns isolation to `builder2`.

Reply:
I think the appliedTo is still Pods in namespace x and this is ingress drop rule, so this should be a policy towards namespace x. Pod C is not really isolated as it can be accessed by and can access everything other than namespace x.

Reply:
Oh right, was thinking incorrectly.